library/core/src/intrinsics.rs

```diff
@@ -1509,12 +1509,14 @@ extern "rust-intrinsic" {
     ///
     /// This intrinsic does not have a stable counterpart.
     #[rustc_nounwind]
+    #[rustc_diagnostic_item = "ptr_unaligned_volatile_load"]
     pub fn unaligned_volatile_load<T>(src: *const T) -> T;
     /// Performs a volatile store to the `dst` pointer.
     /// The pointer is not required to be aligned.
     ///
     /// This intrinsic does not have a stable counterpart.
     #[rustc_nounwind]
+    #[rustc_diagnostic_item = "ptr_unaligned_volatile_store"]
```

Comment:
Those are not in `ptr`, those are intrinsics; are you sure clippy needs those?

Reply:
They're exported in `core::ptr`

Reply:
`unaligned_volatile_{load,store}`? I can't seem to find them in the docs, could you point me to the reexports?